src/messages/GE.json

```diff
@@ -359,11 +359,10 @@
   "As user": "(signed in as user)",
   "Use Graph Explorer as a sample Teams application": "Use Graph Explorer as a sample Teams application",
   "Use Graph Explorer as a signed in user": "Use Graph Explorer as a signed in user",
-  "Resource Specific Consent popup": "To use resource-specific consent permissions, download​",
+  "Resource-specific Consent popup": "To use resource-specific consent permissions, download​",
   "Sample Teams app for Graph Explorer": "Sample Teams app for Graph Explorer",
   "Install sample app": "Install sample app",
   "Learn more about": "Learn more about",
-  "resource specific consent": "resource specific consent",
-  "Graph Explorer Sample App": "Graph Explorer Sample App",
+  "resource-specific consent": "resource specific consent",
```

Comment:
you forgot the second hyphen in the right hand side :)

Reply:
d91f60424212896bb86acf9751c22a1698a89cbb